client/main.go

```diff
@@ -19,24 +19,25 @@ func main() {
 	}
 	reader := bufio.NewReader(conn)
 	serverHandlers := make(map[scs.ActionName]ServerHandleFunc)
-	serverHandlers[scs.GET_ROOMS] = handleGetRooms
-	serverHandlers[scs.POST] = handleMessage
-	serverHandlers[scs.JOIN_ROOM] = handleJoinRoom
-	serverHandlers[scs.LEAVE_ROOM] = handleLeaveRoom
+	serverHandlers[scs.GetRooms] = handleGetRooms
+	serverHandlers[scs.Post] = handleMessage
+	serverHandlers[scs.JoinRoom] = handleJoinRoom
+	serverHandlers[scs.LeaveRoom] = handleLeaveRoom
 	state := &State{state: INIT}
 
 	go handleServerInput(reader, serverHandlers, state)
 	rw := bufio.NewReadWriter(bufio.NewReader(os.Stdin), bufio.NewWriter(conn))
 
-	userHandlers := make(map[ClientState]map[string]UserHandleFunc)
-	initHandlers := make(map[string]UserHandleFunc)
-	initHandlers["create"] = createRoom
-	initHandlers["join"] = joinRoom
-	inRoomHandlers := make(map[string]UserHandleFunc)
-	inRoomHandlers["post"] = post(state)
-	inRoomHandlers["leave"] = leaveRoom
-	userHandlers[INIT] = initHandlers
-	userHandlers[IN_ROOM] = inRoomHandlers
+	userHandlers := map[ClientState]map[string]UserHandleFunc{
+		INIT: {
```

Comment:
Update to use mixed caps instead of all caps.

https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps